# Config
device: "/dev/ttyUSB0"
baudrate: 115200
state_poll_rate: 10.0
goal_push_rate: 60.0
gimbal_mode: 1

